src/agents/onboarding_agent.py

```diff
@@ -124,94 +126,25 @@ def run(self, context: AgentContext) -> Action:  # noqa: C901
             # player.inventory.append(Item(name=name))
             save_game_state(game_state, context)
 
-        if not player.motivation:
```

Comment:
question(non-blocking): why did this get stripped from the onboarding agent? and does this have an impact on the workflow for https://github.com/steamship-core/ai-adventure-agent/pull/25 ?

Reply:
Because it's now no longer up to the player, but rather the game creator, to decide what the ultimate goal of the story is.  There was a chat about this in slack.. the general consensus was to strip down what the player had to provide in this regard, and use the creator's input instead.